src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-snapshots-list/cephfs-subvolume-snapshots-list.component.ts

```diff
@@ -249,8 +250,19 @@ export class CephfsSubvolumeSnapshotsListComponent implements OnInit, OnChanges
   cloneModal() {
     const cloneName = `clone_${moment().toISOString(true)}`;
     const allGroups = Array.from(this.subvolumeGroupList).map((group) => {
```

Comment:
@dnyanee1997 one of the larger issue here is that `this.subvolumeGroupList` only has the list of subvolume groups which has subvolumes present inside it. You can see the filtration from the above codes.
```
this.cephfsSubvolumeService.existsInFs(this.fsName, group.name).pipe(
              switchMap((resp) => {
                if (resp) {
                  this.subvolumeGroupList.push(group.name);
                }
                return of(resp); // Emit the response
              })
            )
```

and this means that even if we fix the `_nogroup` scenario, all the other groups that doesn't have a subvolume is not listed here which shouldn't be the case. this dropdown should have the list of all the groups including the `_nogroup`. Could you look into that too?